app/src/main/java/pl/gittobefit/TrainingStart.java

```diff
@@ -1,15 +1,41 @@
 package pl.gittobefit;
 
+import android.net.Uri;
 import android.os.Bundle;
+
+import androidx.core.view.ViewCompat;
 import androidx.fragment.app.Fragment;
+
+import android.os.CountDownTimer;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
 import android.widget.Button;
+import android.widget.ImageView;
 import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.VideoView;
+
+import java.util.Locale;
+
+public class TrainingStart extends Fragment  {
 
-public class TrainingStart extends Fragment {
+    private static final long START_TIME_IN_MILLIS = 13000;
+    private boolean timerRunning;
+    private long timeLeftInMillis = START_TIME_IN_MILLIS;
+    private CountDownTimer timerAdapter;
+    private TextView timerText;
+    private View view;
+    private ImageView buttonPlay,buttonPause;
+    private LinearLayout exerciseStart,exerciseBackground;
+    private RelativeLayout buttonLayout;
+    private Animation started,exit,downtoup,bright;
+    private Button miss,start,nextExercise;
+    private VideoView videoViewTraining,videoViewStartTraining;
```

Comment:
Po przecinku przydałaby sie spacja:
```suggestion
    private ImageView buttonPlay, buttonPause;
    private LinearLayout exerciseStart, exerciseBackground;
    private RelativeLayout buttonLayout;
    private Animation started, exit, downtoup, bright;
    private Button miss, start, nextExercise;
    private VideoView videoViewTraining, videoViewStartTraining;
```